src/nrfx/hal/nrf_egu.c

```diff
@@ -0,0 +1,176 @@
+/** Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Note that the function prototypes are taken from the NRFx HAL
+ */
+
+ #if 1
+
+#include "hal/nrf_egu.h"
+#include "bs_tracing.h"
+#include "NRF_EGU.h"
+#include <kernel.h>
+#include <irq_ctrl.h>
+
+uint32_t nrf_egu_channel_count(NRF_EGU_Type const *p_reg)
+{
+	return 16;
```

Comment:
Have you checked if nrfx or the mdk defines a value for this?

Reply:
The real nrfx implementation used a macro for every channel, but on nrf52840, all EGUs have 16 channels